packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -39,6 +46,11 @@ const Sublayout = ({
 
   useEffect(() => {
     setMenu({ name: 'default', isVisible: !isWindowSmallInclusive });
+    setToggleMobileView(
+      (location.pathname.includes('discussions') ||
+        location.pathname.includes('search')) &&
+        window.innerWidth <= 425
+    );
```

Comment:
what is the reason that this piece is both here and in other useEffect below? I think this should be only in one place, when the window is getting resized?

Reply:
Wrapped these up